src/runner/PlutoRunner.jl

```diff
@@ -375,6 +375,19 @@ function show_array_row(io::IO, pair::Tuple)
     print(io, "</v></r>")
 end
 
+function show_array_row(io::IO, idxs::AbstractVector{<:Integer}, ary::AbstractArray{<:Any, 1})
```

Comment:
This should be a new function: it does not show a `row`.

Reply:
I can rename it to "rows" but that's too subtle of a difference?